Makefile

```diff
@@ -69,55 +78,70 @@ publish: build lint
 
 .PHONY: clean
 clean: ## Remove intermediate build artifacts & cache.
-	@rm -rf $(BUILDDIR)/*
+	@rm -rf $(BUILDDIR)
```

Comment:
nit: I know it's probably fine to run in force mode, but it still creepy - is there a chance `-r` (recursive) won't work?

Reply:
Removed `-f` in 349343a.